### DISCUSSION TITLE
Strategie around Streamlit

### DISCUSSION THREAD

**Discussion by osi1880vr** [hlky/stable-diffusion-webui#1069] [Q&A]
I just stumbled across the streamlit version of the gui.
I found it while I was reading about animation.

I wonder is there any strategie which gui might win long term? I get a feeling that if you guys fragment the features it gets quite complicated for us users out here ;) 

Dont get me wrong I take what ever you give, I just wonder if there is a plan =)
It would also help me to dig deeper in the right hole to maybe one day I can offer some more code than the one line which I had to steal in the frist place :D

**Comment by Kasiya13** [comment]
Streamlit is currently being worked on, it should be the one getting all the new features once its ready for use.
